sdk/keyvault/azkeys/README.md

```diff
@@ -361,7 +361,7 @@ func main() {
 		panic(err)
 	}
 
-	pager := client.NewListPropertiesOfKeysPager(nil)
+	pager := client.ListPropertiesOfKeys(nil)
```

Comment:
I guess this API still needs renaming?

Reply:
No, this is evidence of me bungling a merge 😊